locales/en-US/app.ftl

```diff
@@ -306,6 +368,57 @@ ProfileLoaderAnimation--loading-message-view-not-found =
 ProfileRootMessage--title = { -profiler-brand-name }
 ProfileRootMessage--additional = Back to home
 
+## ServiceWorkerManager
+
+ServiceWorkerManager--installing-button = Installing…
+ServiceWorkerManager--pending-button = Apply and reload
+ServiceWorkerManager--installed-button = Reload the application
+ServiceWorkerManager--updated-while-not-ready =
+    A new version of the application was applied before this page
+    was fully loaded. You might see malfunctions.
+ServiceWorkerManager--new-version-is-ready =
+    A new version of the application has been downloaded and is ready to use.
+ServiceWorkerManager--hide-notice-button =
+    .title = Hide the reload notice
+    .aria-label = Hide the reload notice
```

Comment:
These are the same here. I wanted to have a variable or a message reference here at first, but when I looked at the firefox source docs, that was discouraged. See: https://firefox-source-docs.mozilla.org/l10n/fluent/review.html#message-references

Reply:
Within the same message you could still use it, but it really doesn't help (quite the opposite). So, good as it is 👍🏼